core/src/main/java/hudson/PluginManager.java

```diff
@@ -24,6 +24,7 @@
 package hudson;
 
 import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.security.ACLContext;
```

Comment:
the other imports are not alphabetically sorted, we should have a PR to fix them all up